content/applications/marketing/events/revenues_report.rst

```diff
@@ -0,0 +1,161 @@
+- :guilabel:`Published Events`: select this option to show revenue-related data for published
+  events.
+- :guilabel:`Add Custom Filter`: create a custom filter to analyze event-related revenue data. To
+  learn more, refer to the documentation on :ref:`custom filters <search/custom-filters>`.
+
+Group By options
+~~~~~~~~~~~~~~~~
+
+In the :guilabel:`Group By` column of the drop-down mega menu, there are various event-related
+options that can be utilized to create custom groupings of data for reports.
+
+Multiple :guilabel:`Group By` options can be selected at once.
+
+The :guilabel:`Group By` column has the following options:
+
+- :guilabel:`Event Type`: group data based on the type of event.
+- :guilabel:`Event`: organize data into individual groups, separated by events.
+- :guilabel:`Product`: group data based on the event registration product.
+- :guilabel:`Ticket`: group data based on the type of event ticket purchased by attendees.
+- :guilabel:`Registration Status`: group data based on the status of registrations.
+- :guilabel:`Sale Order Status`: group data based on the status of event-related sales orders.
+- :guilabel:`Customer`: group data based on customer records.
+- :guilabel:`Add Custom Group`: click the :icon:`fa-caret-down` :guilabel:`(down arrow)` icon to
+  reveal a drop-down of grouping options to choose from. To select one, simply click on the desired
+  option from the drop-down menu, and Odoo adds it to the :guilabel:`Group By` column. Multiple
+  selections can be made.
+
+Sample report: event ticket analysis
+====================================
+
+The following is an example of how various filters and grouping options can be utilized to create a
+useful analytic report related to event revenues. In this case, the configurations present data
+about sold or free tickets to published events, with the metrics separated by ticket type and event.
+
+.. image:: revenues_report/event-ticket-analysis.png
+   :align: center
+   :alt: The event ticket analysis sample report with unique filters and groupings in place.
+
+To create such a report, stay in the default graph view, but remove the default filters from the
+:guilabel:`Search...` bar.
+
+Then, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` to the right of the
+:guilabel:`Search...` bar, to reveal the drop-down mega menu of filter and grouping options.
+
+From here, select the :guilabel:`Free` and :guilabel:`Sold` from the :guilabel:`Filters` column.
+
+Then, since it is desired to **only** view data related to already published events, select the
+:guilabel:`Published Events` option in the :guilabel:`Filters` column, as well.
+
+Next, in the :guilabel:`Group By` column, select the :guilabel:`Event` and :guilabel:`Ticket`
+options, in that sequential order. Doing so first groups the data by event *then* by ticket type,
+and not the other way around, which provides a more useful array of data to analyze.
```

Comment:
```suggestion
options, in that sequential order. Doing so first groups the data by event *then* by ticket type, which provides a more useful array of data to analyze.
```
It may be worth a tip block about how the order of groups are selected impacts the results. (I realize this falls under more of the "essentials" docs, but I don't think it is a problem to add a quick mention of it).